tests/unit_test/data_processor/data_processor2_test.py

```
@@ -0,0 +1,49 @@
+import os
+from kairon.shared.utils import Utility
+os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)